backend/src/application/repositories/chat-room-repository.ts

```diff
@@ -0,0 +1,30 @@
+import { ChatRoom } from '@/domain';
+
+export namespace ChatRoomRepository {
+  export const Name = 'ChatRoomRepository';
+
+  export namespace Create {
+    export type Input = ChatRoom;
+
+    export type Output = ChatRoom;
+  };
+
+  export namespace GetMessages{
+    export type Input = { chatRoomId: string, };
+
+    export type Output = Pick<ChatRoom, 'messages'>;
```

Comment:
Caso não encontre retornará null
```suggestion
    export type Output = Pick<ChatRoom, 'messages'> | null;
```